AdminUi/apps/admin_ui/lib/home/identity_details/identity_relationships/identity_relationship_source.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:admin_api_sdk/admin_api_sdk.dart';
+import 'package:admin_api_types/admin_api_types.dart';
+import 'package:data_table_2/data_table_2.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:intl/intl.dart';
+import 'package:logger/logger.dart';
+
+class IdentityRelationshipSource extends AsyncDataTableSource {
```

Comment:
all other data sources are called ....DataSource, this is only called ...Source. Would you please give an explanation why this is named like that?